# Header
std_msgs/Header header
string name
int32 id
HMCL_SPATdata[] States

